src/Plugins/MPTTrie/Cryptography/MPTTrie/Trie.Find.cs

```diff
@@ -47,7 +46,7 @@ private ReadOnlySpan<byte> Seek(ref Node node, ReadOnlySpan<byte> path, out Node
                             start = node;
                             return ReadOnlySpan<byte>.Empty;
                         }
-                        return Concat(path[..1], Seek(ref node.Children[path[0]], path[1..], out start));
+                        return new([.. path[..1], .. Seek(ref node.Children[path[0]], path[1..], out start)]);
```

Comment:
These type of change maybe should be done in another pr, since they are not related.

Reply:
It is replacing `Helper.Concat` with `C#` Concat collection.

Reply:
Agree, I'm not sure if this is faster, this change is not related to this PR

Reply:
It should be faster it gets compiled in at build. I think it related, because I said `Fixed problems and warnings that occurred. Along with naming violations.`